### ISSUE TITLE
Email component_name variable not showing

### ISSUE THREAD

**Issue by CandyManN** [CachetHQ/Cachet#2034]
- [X] I am running the [latest release](https://github.com/CachetHQ/Cachet/releases/latest) version of Cachet.
- [X] I am running at least PHP 5.5.9. *You can check this by running `php -v`.*
- [X] I have ran `rm -rf bootstrap/cache/*`.

Basically, when I change a status of a component, the email to the subscribers will appear like that: 

"The component component_name has seen a status change."

There isn't written the component name.

The text-logo of the mail seems to be broken because it's not showing up the css, but only the text.
The other links in the mail works without problems.
No errors in any log file so far.

**Comment by jbrooksuk** [comment]
What version of Cachet are you using?

**Comment by CandyManN** [comment]
I'm using the v2.3.8

**Comment by jbrooksuk** [comment]
Thanks @CandyManN I'll get this fixed ASAP.

**Comment by jbrooksuk** [comment]
Out of interest, what language are you using? I see no reason why this isn't working in the English translation.

**Comment by jbrooksuk** [comment]
That said, looking through the translations I've found that Italian is broken.

**Comment by CandyManN** [comment]
Yeah, it's the Italian one :+1:

**Comment by jbrooksuk** [comment]
Awesome this will be fixed in v2.3.9. I'm just waiting on a test case from someone which fixes another issue and then I can ship.
